src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +36,10 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.range(0, numbers.size())
+                .map(number -> number % 2 == 1 ? numbers.get(number) - 1 : numbers.get(number))
+                .filter(number -> number % 2 == 1)
+                .average().orElseThrow(NoSuchElementException::new);
```

Comment:
```suggestion
                .average()
                .getAsDouble;
```
you can use getAsDouble